geode-core/src/test/java/org/apache/geode/internal/cache/partitioned/rebalance/PartitionedRegionLoadModelJUnitTest.java

```diff
@@ -443,7 +443,7 @@ public void testIncompleteColocation() throws UnknownHostException {
    * lmm, it will prevent a bucket move
    */
   @Test
-  public void testColocationEnforceLocalMaxMemory() throws UnknownHostException {
+  public void testColocationTwoNoneEvictionRegions() throws UnknownHostException {
```

Comment:
I would rename the test case to: `testColocationTwoNonEvictionRegionsEnforceLocalMaxMemory()`